### PR TITLE
PV2-1193: sorting out writing payments to DB

### PR DESCRIPTION
**Pull request by kuzinator**
this PR is not ready to review. It's here to trigger DSI build with more logging.

### CODE REVIEW

File: src/SFA.DAS.Payments.Audit.Application/Data/PaymentsEventModelDataTable.cs [hunk 1]
```diff
@@ -27,9 +27,9 @@ protected PaymentsEventModelDataTable()
                 new DataColumn("LearningAimPathwayCode"),
                 new DataColumn("LearningAimFundingLineType"),
                 new DataColumn("Ukprn"),
-                new DataColumn("IlrSubmissionDateTime"),
+                new DataColumn("IlrSubmissionDateTime", typeof(DateTime)),
```
Comment (samuel-dev-dba) [inline src/SFA.DAS.Payments.Audit.Application/Data/PaymentsEventModelDataTable.cs]: Is this not supposed to be - DateTimeOffset as defined in Table Column DataType --  [IlrSubmissionDateTime] DATETIME2 NOT NULL,

Comment (kuzinator) [reply src/SFA.DAS.Payments.Audit.Application/Data/PaymentsEventModelDataTable.cs]: no, DateTimeOffset in .NET is represented by DATETIMEOFFSET in SQL. DateTime is DATETIME2

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.Audit.Application/Data/PaymentsEventModelDataTable.cs]: Ha  my mistake

**Comment by ESFA-Pawan** [comment]
Release passed in DAS and DPT - DAS Payments V2-0.0.2582-R001

**Pull request merged**